WalletWasabi.Documentation/FAQ.md

```diff
@@ -0,0 +1,178 @@
+FAQ for [Wasabi Wallet](https://github.com/zkSNACKs/WalletWasabi)
+By [6102bitcoin](https://twitter.com/6102bitcoin)
+
+Note: Many of these Q&A have been copied from real users, see footer for acknowledgements. 
+
+# Index
+- Pre-Install
+- Install
+- Pre-Mix
+- Mixing
+- Post-Mix
+- Meta
+- Acknowledgements
+
+# Pre-Install
+
+### Who is behind Wasabi?
+
+The Company that is developing Wasabi is zkSNACKs LTD ([twitter](@Zksnacks_LTD) | [website](https://zksnacks.com/))  
+Members as of 2019.05.08:
+
+| Name              | Twitter                                                 | Position			   |
+| ------------------| --------------------------------------------------------|	-----------------------|
+| Gergely Hajdú 	| [@gergely_hajdu](https://twitter.com/gergely_hajdu) 	  | Co-CEO & Co-Founder	   |
+| Bálint Harmat		| [@bharmat84](https://twitter.com/bharmat84) 			  | Co-CEO & Co-Founder    |
+| Ádám Ficsór		| [@nopara73](https://twitter.com/nopara73)	              | Developer & Co-Founder |
+| Lucas Ontivero	| [@lontivero](https://twitter.com/)                      | Developer & Co-Founder |
+| Dan Walmsley   	| ([@danwalmsley](https://github.com/danwalmsley))        | Developer              |
+| Dávid Molnár   	| ([@molnardavid84](https://twitter.com/molnardavid84))   | Developer              |
+
+# Install
+
+### How do I install Wasabi?
+
+Follow [this guide](https://github.com/zkSNACKs/WalletWasabi/blob/master/WalletWasabi.Documentation/Guides/InstallInstructions.md)
+
+If you use Windows there is a [video guide](https://youtu.be/D8U53PFEsVk).
+
+### Do I need to run Tor?
+
+All Wasabi network traffic goes via Tor by default - no need to set up Tor yourself. If you do already have Tor, and it is running, then Wasabi will try to use that first.  
+
+# Pre-Mix
+
+### My wallet can't send to Bech32 addresses - what wallets can I use instead? 
+
+Wasabi generates Bech32 addresses only. These addresses start with the characters bc1... Some wallets/exchanges don't yet support this type of address and my give an error message (e.g. "unknown bitcoin address"). The solution is to manage your funds with a wallet which does support Bech32. Of [all the wallets](https://en.bitcoin.it/wiki/Bech32_adoption) that support Bech32, only the following wallets have Coin Control;
+
+| Platform 						| Recommended 																					| Other										|
+| ---------						| -----------------------------------------------------	|------------------------	|
+| Mobile (Android)		| [Samourai](https://samouraiwallet.com/)								| [Electrum](https://play.google.com/store/apps/details?id=org.electrum.electrum)								|
+| Desktop 						| [Bitcoin Core](https://bitcoincore.org/en/download/)	| [Electrum](https://electrum.org)	|
+
+I am not aware of any wallets for iOS which support coin control. If you are aware of any other Wallets with coin control please make a pull request.
```

Comment:
While I see why you think coin control is important here, I think it's not that important here. Without going into the details, this section would need maintenance, so I'd vote for removing the end of it. The link to bech32 adoption however should be present.

Reply:
Removed.